SpatialGDK/Extras/spot.version

```diff
@@ -1 +1 @@
-20190822.155324.b3f16cc67f
\ No newline at end of file
+20191029.144741.87a7d78768
```

Comment:
Is the lack of newline here going to break things?

Reply:
I was able to local and cloud deploy so I think not?

Reply:
chatted offline, non issue